pkgs/servers/monitoring/pyroscope/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "pyroscope";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "grafana";
+    repo = "pyroscope";
+    hash = "sha256-a59lD7S14n8VaUWGCLKhmHlB8h/mADUKtEn/25j+I+M=";
+  };
+
+  proxyVendor = true;
+  vendorHash = "sha256-Ncx1fe2s9VvlE0CUUnn/NzX9CO7csB/JmIEXY5+2ZgQ=";
+
+  ldflags = let
+    prefix = "github.com/grafana/pyroscope/pkg/util/build";
+  in [
+    "-s" "-w"
+    # https://github.com/grafana/pyroscope/blob/v1.0.0/Makefile#L30
+    "-X ${prefix}.Version=${version}"
+    "-X ${prefix}.Branch=v${version}"
+    "-X ${prefix}.Revision=v${version}"
+    "-X ${prefix}.BuildUser=nix"
+    "-X ${prefix}.BuildDate=1980-01-01T00:00:00Z"
+  ];
+
+  subPackages = [
+    "cmd/pyroscope"
+    "cmd/profilecli"
+  ];
+
+  meta = with lib; {
+    description = "Grafana Pyroscope is an open source database that provides fast, scalable, highly available, and efficient storage and querying of profiling data.";
+    license = licenses.agpl3;
+    homepage = "https://grafana.com/oss/pyroscope";
```

I think we might as well add a `changelog`:

```suggestion
    homepage = "https://grafana.com/oss/pyroscope";
    changelog = "https://github.com/grafana/pyroscope/releases/tag/v${version}";
```